commons/push/integration/garmin/garmin_push_user_metrics.avsc

```diff
@@ -0,0 +1,14 @@
+{
+  "namespace": "org.radarcns.push.integration.garmin",
+  "name": "GarminUserMetrics",
+  "type": "record",
+  "doc": "User Metrics are per-user calculations performed by Garmin based on the underlying data uploaded from the user’s device. This data can be specific to a single device and field availability is dependent on device model support. Unlike other summaries, User Metrics are associated only with a calendar date, not a specific time frame, and only the most recent value for any fields is presented to the user. Each metric directly corresponds to the similarly named field found in Garmin Connect.",
+  "fields": [
+    {"name": "summaryId", "type": "string", "doc": "Unique identifier for the summary."},
+    {"name": "time", "type": "double", "doc": "Start time of the activity in seconds since January 1, 1970, 00:00:00 UTC (Unix timestamp)."},
+    {"name": "timeReceived", "type": "double", "doc": "Time that this record was collected by a service in seconds since the Unix Epoch (s)."},
+    {"name": "calendarDate", "type": ["null", "string"], "doc": "The calendar date this summary would be displayed on in Garmin Connect. The date format is ‘yyyy-mm-dd’.", "default": null},
```

Comment:
Just `date` would also be fine. Please clarify in doc that this is the date in the local time zone of the garmin connect.